lib/plugins/jexlParser.js

```diff
@@ -53,20 +53,23 @@ jexl.addTransform('urldecode', (val) => decodeURI(val));
 jexl.addTransform('replacestr', (str, from, to) => str.replace(from, to));
 jexl.addTransform('replaceregexp', (str, reg, to) => str.replace(new RegExp(reg), to));
 jexl.addTransform('replaceallstr', (str, from, to) => str.replaceAll(from, to));
-jexl.addTransform('replaceallregexp', (str, reg, to) => str.replaceAll(new RegExp(reg,"g"), to));
+jexl.addTransform('replaceallregexp', (str, reg, to) => str.replaceAll(new RegExp(reg, 'g'), to));
 jexl.addTransform('split', (str, ch) => str.split(ch));
 jexl.addTransform('mapper', (val, values, choices) => choices[values.findIndex((target) => target === val)]);
 jexl.addTransform(
     'thmapper',
     (val, values, choices) =>
-        choices[values.reduce((acc,curr,i,arr) => (acc===0)||acc?acc:val<=curr?acc=i:acc=null,null)],
+        choices[
+            values.reduce(
+                (acc, curr, i, arr) => (acc === 0 || acc ? acc : val <= curr ? (acc = i) : (acc = null)),
+                null
+            )
+        ]
 );
-jexl.addTransform(
-    'bitwisemask',
-    (i,mask,op) => op==="&"?parseInt(i)&mask: op==="|"?parseInt(i)|mask: op==="^"?parseInt(i)^mask:i
+jexl.addTransform('bitwisemask', (i, mask, op) =>
+    op === '&' ? parseInt(i) & mask : op === '|' ? parseInt(i) | mask : op === '^' ? parseInt(i) ^ mask : i
```

Comment:
Need to be sync in https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/expressionLanguage.md#available-functions or they are just indent/linter changes?

Reply:
just indent / linter auto changes

Reply:
NTC